src/Claims.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * Standard OpenID Connect address claim.
+ * The Address Claim represents a physical mailing address.
+ *
+ * @internal
+ * @see https://openid.net/specs/openid-connect-core-1_0.html#AddressClaim
+ */
+export interface OidcAddressClaim {
+    /** Full mailing address, formatted for display or use on a mailing label. This field MAY contain multiple lines, separated by newlines. Newlines can be represented either as a carriage return/line feed pair ("\r\n") or as a single line feed character ("\n"). */
+    formatted?: string;
+    /** Full street address component, which MAY include house number, street name, Post Office Box, and multi-line extended street address information. This field MAY contain multiple lines, separated by newlines. Newlines can be represented either as a carriage return/line feed pair ("\r\n") or as a single line feed character ("\n"). */
+    street_address?: string;
+    /** City or locality component. */
+    locality?: string;
+    /** State, province, prefecture, or region component. */
+    region?: string;
+    /** Zip code or postal code component. */
+    postal_code?: string;
+    /** Country name component. */
+    country?: string;
+}
+
+/**
+ * Standard OpenID Connect claims.
+ * They can be requested to be returned either in the UserInfo Response or in the ID Token.
+ *
+ * @internal
+ * @see https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
+ */
+export interface OidcClaims {
+    [customClaimKey: string]: unknown;
```

Comment:
I think `OidcStandardClaims` would be a better name for this - to me "ID Token claims" are synonymous with "OIDC claims", so it's a bit confusing to see this interface not contain the other required claims like "sub", "aud", etc.

Additionally, this `customClaimKey` index type feels out of place here especially since you already redefine in the `IdTokenClaims` interface.

Reply:
1) Done
2) Agreed. Done